atm.js

```diff
@@ -1,21 +1,69 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//Checking Deposit
+  $(".cdepositButton").on("click",function(){
+    console.log("i clicked the deposit button")
+    var deposit = parseInt($(".input").val())
+    //console.log(deposit)
+   var balance=$(".cBalance").text()
+   //console.log(balance)
+   balance= parseInt(balance.replace("$", ""))
+  var newBalance= deposit + balance
+ //console.log(balance)
+ $(".cBalance").text("$" + newBalance)
+ //I wasn't sure what instructions meant regarding changing the color to reflect the balance
+ if (newBalance >= 500){
+ $(".cBalance").css("background-color", "#6df15d")
+ }
 
-}
 
-function checkingWithdrawal(amount){
+ })
 
+//Checking Withdrawal
+$(".cwithdrawalButton").on("click",function(){
+  console.log("i clicked the withdrawal button")
+var withdrawal = parseInt($(".input").val())
+var balance=$(".cBalance").text()
+if (withdrawal > balance) { //unable to figure out the code to ignore transaction if withdrawal > deposit total 
```

Comment:
does balance need to be parse as an integer here? not sure if this is related to the `withdrawal > deposit` issue